apps/posts/src/components/chart/CustomTooltipContent.tsx

```diff
@@ -0,0 +1,39 @@
+import {formatDisplayDate} from '@tryghost/shade';
```

Comment:
Again, this is a temporary component that will reused instead of copied over from the `stats` app once I merged React apps.